Demo/src/main/java/com/braintreepayments/demo/DemoActivity.java

```diff
@@ -48,16 +48,6 @@ protected void onCreate(Bundle savedInstanceState) {
         registerSharedPreferencesListener();
     }
 
-    @Override
-    protected void onResume() {
-        super.onResume();
-
-        if (BuildConfig.DEBUG && ContextCompat.checkSelfPermission(this, WRITE_EXTERNAL_STORAGE)
-                != PackageManager.PERMISSION_GRANTED) {
-            ActivityCompat.requestPermissions(this, new String[]{WRITE_EXTERNAL_STORAGE}, 1);
-        }
-    }
-
```

Comment:
This code causes an infinite `onResume()` loop on my test device. The write permission isn't a part of the SDK we don't need storage permissions.